source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+.. _`fmt cdiff demux`:
+
+Demultiplexing sequences
+------------------------
+
+To demultiplex sequences we need to know which barcode sequence is associated with each sample. This information is contained in the `sample metadata`_ file. You can run the following commands to demultiplex the sequences (the ``demux emp-paired`` command refers to the fact that these sequences are barcoded according to the `Earth Microbiome Project`_ protocol, and are pared-end reads). The ``demux.qza`` QIIME 2 artifact will contain the demultiplexed sequences. Additionally, we are passing the parameter ```--p-rev-comp-mapping-barcodes```, reverse complements the barcode sequences in the sample metadata prior to demultiplexing.
+
+.. command-block::
+
+    qiime demux emp-paired \
+      --i-seqs emp-paired-end-sequences.qza \
+      --m-barcodes-file sample-metadata.tsv \
+      --m-barcodes-column BarcodeSequence \
+      --o-per-sample-sequences demux.qza \
+      --p-rev-comp-mapping-barcodes
+
+After demultiplexing, it's useful to generate a summary of the demultiplexing results. This allows you to determine how many sequences were obtained per sample, and also to get a summary of the distribution of sequence qualities at each position in your sequence data.
+
+.. command-block::
+
+    qiime demux summarize \
+      --i-data demux.qza \
+      --o-visualization demux.qzv
+
+.. note::
+   All QIIME 2 visualizers (i.e., commands that take a ``--o-visualization`` parameter) will generate a ``.qzv`` file. You can view these files with ``qiime tools view``. We provide the command to view this first visualization, but for the remainder of this tutorial we'll tell you to *view the resulting visualization* after running a visualizer, which means that you should run ``qiime tools view`` on the .qzv file that was generated.
+
+   .. command-block::
+      :no-exec:
+
+      qiime tools view demux.qzv
+
+   Alternatively, you can view QIIME 2 artifacts and visualizations at `view.qiime2.org <https://view.qiime2.org>`__ by uploading files or providing URLs. There are also precomputed results that can be viewed or downloaded after each step in the tutorial. These can be used if you're reading the tutorial, but not running the commands yourself.
+
+Sequence quality control and feature table construction
+-------------------------------------------------------
+
+QIIME 2 plugins are available for several quality control methods, including `DADA2`_, `Deblur`_, and `basic quality-score-based filtering`_. In this tutorial we present this step using `DADA2`_ and `Deblur`_. These steps are interchangeable, so you can use whichever of these you prefer. The result of both of these methods will be a ``FeatureTable[Frequency]`` QIIME 2 artifact, which contains counts (frequencies) of each unique sequence in each sample in the dataset, and a ``FeatureData[Sequence]`` QIIME 2 artifact, which maps feature identifiers in the ``FeatureTable`` to the sequences they represent.
```

Comment:
Please link to the overview tutorial and/or stress that readers should refer to the original publications, the overview tutorial, and possibly other literature to learn more about denoising and the differences between these methods.

It may also be worth mentioning that denoising takes the place of OTU clustering, and so ASVs do not need to be clustered. Many users get hung up on this... even if you explain in more detail below, it would be worth a brief mention here — the overview tutorial goes into more detail so can just reference that.